### PR TITLE
RAIN-1258 : user-events : README.md and LOCALSETUP.md

### PR DESCRIPTION
**Pull request by shreyab30-eGov**

### CODE REVIEW

File: egov-user-event/LOCALSETUP.md [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+  - [X] Consumer
+  - [X] Producer
+
+## Running Locally
+
+To run the egov-user-event locally, you need to port forward below services locally
+
+```bash
+function kgpt(){kubectl get pods -n egov --selector=app=$1 --no-headers=true | head -n1 | awk '{print $1}'}
+kubectl port-forward -n egov $(kgpt egov-persister) 8082:8080
```
Comment (tarunlalwani) [inline egov-user-event/LOCALSETUP.md:22]: Why persister needs to be port forwarded?

File: egov-user-event/LOCALSETUP.md [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+kubectl port-forward -n egov $(kgpt egov-localisation) 8087:8080
+kubectl port-forward -n egov $(kgpt egov-mdms-service) 8094:8080
+```
+
+To run the th-services locally, update below listed properties in `application.properties` prior to running the project:
+
+```ini
+# Default pagination offset
+mseva.notif.search.offset=
```
Comment (tarunlalwani) [inline egov-user-event/LOCALSETUP.md:31]: These properties are not required from local setup perspective and should be in readme only?

File: egov-user-event/LOCALSETUP.md [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+
+# Default pagination limit
+mseva.notif.search.limit=
```
Comment (tarunlalwani) [inline egov-user-event/LOCALSETUP.md:34]: There are no hosts properties even though we have specified port forwarding?

**Review by tarunlalwani — CHANGES_REQUESTED** [review]

File: egov-user-event/LOCALSETUP.md (hunk 3 above)
Comment (rohit-eGov) [reply egov-user-event/LOCALSETUP.md]: Added host properties in localsetup.md

File: egov-user-event/LOCALSETUP.md (hunk 2 above)
Comment (rohit-eGov) [reply egov-user-event/LOCALSETUP.md]: described the above properties in readme.md

File: egov-user-event/LOCALSETUP.md (hunk 1 above)
Comment (rohit-eGov) [reply egov-user-event/LOCALSETUP.md]: Updated the above command

**Review by tarunlalwani — APPROVED** [review]

**Pull request merged**